database/create_tbl.py

```diff
@@ -0,0 +1,26 @@
+from time import sleep
+
+import pymysql
+from database.connection import db_connect
+
+def db_createtbl(db):
+    cursor=db.cursor()
+
+
+    studentRecord = """CREATE TABLE tbl_employee1(  
```

Comment:
This should be defined as constant outside this function but in the same file. Should name this constant like 
**_EMPLOYEE_TABLE_**